sdk/resourcemanager/Proto.Client/compute/AvailabilitySetBuilder.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.ResourceManager.Core;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Proto.Compute
```

Comment:
why not remove this (and other) Builder class until we have a clearer final design for the builder pattern?